services/github/github-forks.service.js

```diff
@@ -67,11 +73,17 @@ module.exports = class GithubForks extends GithubAuthService {
   }
 
   async handle({ user, repo }) {
-    const { forks_count: forkCount } = await this._requestJson({
-      url: `/repos/${user}/${repo}`,
+    const json = await this._requestGraphql({
+      query:
+        'query ($user: String!, $repo: String!) { repository(owner: $user, name: $repo) { forks { totalCount } } }',
```

Comment:
To make this easier to read, what do you think about using a backquote string with line breaks?

Related, would using the `gql` tagged template literal, and passing a parsed query into `requestGraphql` allow us to parse this once when the code is parsed instead of on every request?